onnxruntime/contrib_ops/contrib_kernels_cuda.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can this file be present in the cuda subdirectory? For our regular ops, we keep the file that registers all the ops (cuda/cuda_execution_provider.cc/h) in the same directory where the kernels are present. Would be good to remain consistent.

Reply:
moved